Makefile

```
@@ -29,11 +29,34 @@ LDFLAGS := -s -w -extldflags '-static'
 
 SRCFILES := $(shell find . ! -path './out/*' ! -path './.git/*' -type f)
 
+ALL_ARCH=amd64 arm arm64 ppc64le s390x
+ML_PLATFORMS=linux/amd64,linux/arm,linux/arm64,linux/ppc64le,linux/s390x
 ALL_BINARIES ?= $(addprefix out/configmap-reload-, \
-									$(addprefix linux-,amd64 ppc64le s390x arm64) \
+									$(addprefix linux-,amd64 ppc64le s390x arm arm64) \
 									darwin-amd64 \
 									windows-amd64.exe)
 
+ifeq ($(GOARCH),amd64)
+	BASEIMAGE?=busybox
```

I'd prefer to replace the `ifeq` blocks with:

```suggestion
DEFAULT_BASEIMAGE_amd64   := busybox
DEFAULT_BASEIMAGE_arm     := armhf/busybox
DEFAULT_BASEIMAGE_arm64   := aarch64/busybox
DEFAULT_BASEIMAGE_ppc64le := ppc64le/busybox
DEFAULT_BASEIMAGE_s390x   := s390x/busybox

BASEIMAGE ?= $(DEFAULT_BASEIMAGE_$(GOARCH))

BINARY=configmap-reload-linux-$(GOARCH)
```